pkg/docker/config/config.go

```diff
@@ -136,7 +136,15 @@ func getAuthFilePaths(sys *types.SystemContext) []authPath {
 		// Logging the error as a warning instead and moving on to pulling the image
 		logrus.Warnf("%v: Trying to pull image in the event that it is a public image.", err)
 	}
+
+	var xdgCfgHomePath string
+	if xdgCfgHome := os.Getenv("XDG_CONFIG_HOME"); xdgCfgHome != "" {
+		xdgCfgHomePath = filepath.Join(xdgCfgHome, xdgRuntimeDirPath)
```

Comment:
Please introduce a new `xdcConfigHomePath` constant, or maybe somehow rename the old one; don’t just silently reuse it for a different purpose.

Reply:
Looking at the other comment — this should probably end up as a “get XDG_CONFIG_HOME path” + “append a path relative to XDG_CONFIG_HOME”.